### ISSUE TITLE
Symbolic representation issue

### ISSUE THREAD

**Issue by xueyagaga** [alipay/Pyraformer#17]
Great work!
However, the representation on Page 5:
![image](https://user-images.githubusercontent.com/64541307/199386587-4c0bb18f-76b5-449d-80b4-9827d617fdb4.png)

is not similar to Table 4:
![image](https://user-images.githubusercontent.com/64541307/199386501-b0639d80-903f-4e43-88e7-24615bc4e270.png)

I thought the latter ones were correct.
Thank you!

**Comment by Zhazhan** [comment]
Yes, the expression in Table 4 is correct. Thank you for pointing this out!
